course/src/isometry.cc

```diff
@@ -276,6 +271,16 @@ Matrix3& Matrix3::operator = (Matrix3&& m) {
     return *this;
 }
 
+Matrix3 Matrix3::operator ^ (const Matrix3& m) const { 
```

Comment:
What does this do? This does not look necessary.

Reply:
It's the matrix product. The operator *  is an element-wise multiplication